modules/mysql/src/test/java/org/testcontainers/junit/mysql/SimpleMySQLTest.java

```diff
@@ -234,8 +240,43 @@ public void testWithAdditionalUrlParamInJdbcUrl() {
         }
     }
 
+    @Test
+    public void testWithOnlyUserReadableCustomIniFile() throws Exception {
```

Comment:
Should I be able to reproduce this setting umask with 027, right?

Reply:
Yeah, the test kinda only an "emulation" but I tested my real world setup with this fix using `umask 0027`.

Reply:
PS: I also did run this new test before and after the fix; before the fix it failed and now it passes.